x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_histogram_panel/helpers.tsx

```diff
@@ -181,3 +181,15 @@ export const createResetGroupByFieldAction: CreateResetGroupByFieldAction = ({
   },
   order,
 });
+
+export const createGenericSubtitle = (
+  isInitialLoading: boolean,
+  showTotalAlertsCount: boolean,
+  totalAlerts: string
+) => !isInitialLoading && showTotalAlertsCount && totalAlerts;
+
+export const createEmbeddedDataSubtitle = (
+  embeddedDataLoaded: boolean,
+  embeddedDataAvailable: boolean,
+  totalAlerts: string
+) => embeddedDataLoaded && embeddedDataAvailable && totalAlerts;
```

Comment:
small details but I feel like having these one-liner functions extracted in a separate file make the original file harder to read...
I'm not saying I prefer having the if/else condition directly in the other file, but I feel like moving some code in those helpers file is really useful if the function is reusable or if it's big enough that it was making the original place much harder to read.
Also moving them out would imply adding unit test, which in this case seems way overkill...